shared/js/background/utils.es6.js

```diff
@@ -327,15 +328,15 @@ function isSameTopLevelDomain (url1, url2) {
  *
  * @param {String} featureName - the name of the feature
  * @param {String} customState - An optional custom state to check for
- * @returns {bool} - if feature is enabled/matches specified state
+ * @returns {bool} - if feature is enabled
  */
-function isFeatureEnabled (featureName, customState) {
+function isFeatureEnabled (featureName) {
```

Comment:
Removed this feature part as we have the ability in the config but we call this method in for loops within this utils; there's a clear logic problem there and would need a rethink before using it in this manner.